src/aind_metadata_service/sharepoint/client.py

```diff
@@ -1295,42 +1308,559 @@ def _map_list_item_to_head_frame(
+                )
+                if injection_type.strip() == injection_types.IONTO.value:
+                    injection_current = parse_str_into_float(
+                        list_item.get_property(list_fields.INJ4_CURRENT.value)
+                    )
+                    alternating_current = list_item.get_property(
+                        list_fields.INJ4_ALTERNATING_TIME.value
+                    )
+                    injection_duration = list_item.get_property(
+                        list_fields.INJ4_IONTO_TIME.value
+                    )
+                    injection = IontophoresisInjection.construct(
+                        start_date=start_date,
+                        end_date=end_date,
+                        experimenter_full_name=experimenter_full_name,
+                        iacuc_protocol=iacuc_protocol,
+                        animal_weight_prior=animal_weight_prior,
+                        animal_weight_post=animal_weight_post,
+                        procedure_type=injection_type,
+                        workstation_id=workstation_id,
+                        injection_hemisphere=burr_hemisphere,
+                        injection_coordinate_ml=burr_coordinate_ml,
+                        injection_coordinate_ap=burr_coordinate_ap,
+                        injection_coordinate_depth=burr_coordinate_depth,
+                        injection_angle=burr_angle,
+                        injection_current=injection_current,
+                        injection_duration=injection_duration,
+                        alternating_current=alternating_current,
+                    )
+                else:
+                    injection_volume = parse_str_into_float(
+                        list_item.get_property(
+                            list_fields.INJ4VOLPERDEPTH.value
+                        )
+                    )
+                    injection = NanojectInjection.construct(
+                        start_date=start_date,
+                        end_date=end_date,
+                        experimenter_full_name=experimenter_full_name,
+                        iacuc_protocol=iacuc_protocol,
+                        animal_weight_prior=animal_weight_prior,
+                        animal_weight_post=animal_weight_post,
+                        procedure_type=injection_type,
+                        workstation_id=workstation_id,
+                        injection_hemisphere=burr_hemisphere,
+                        injection_coordinate_ml=burr_coordinate_ml,
+                        injection_coordinate_ap=burr_coordinate_ap,
+                        injection_coordinate_depth=burr_coordinate_depth,
+                        injection_angle=burr_angle,
+                        injection_volume=injection_volume,
+                    )
+                burr_4_procedures.append(injection)
+            elif procedure.strip() == nsb_burr_types.FIBER_IMPLANT.value:
+                name = ProbeName.PROBE_D.value
```

Comment:
Probe_D isn't defined anywhere

Reply:
it's defined in aind-data-schema 0.12.13, did you run pip install -e .[dev]

Reply:
My bad, it looks like it's defined in the updated aind-data-schema